global-hooks/openapi/config-values.yaml

```diff
@@ -156,6 +179,7 @@ properties:
             type: object
             additionalProperties: false
             description: |
+              !!! IT IS DEPRECATED AND WILL BE REMOVED IN THE FUTURE !!!
```

Comment:
There is no need to add such an exclamation. We have a mean to mark the field as deprecated. 

```yaml
masterNode:
  x-doc-deprecated: true
  ...
```

Reply:
Thanks, i did not know this